internal/commands/push/inputs_test.go

```diff
@@ -34,6 +34,32 @@ func TestPushInputsResolve(t *testing.T) {
 		})
 	})
 
+	t.Run("should return an error when specified local path does not exist", func(t *testing.T) {
+		profile, teardown := mock.NewProfileFromTmpDir(t, "app_init_input_test")
+		defer teardown()
+
+		localPath := "fakePath"
+
+		i := inputs{LocalPath: localPath}
+		assert.Equal(t, errProjectInvalid{Path: localPath, Exists: false}, i.Resolve(profile, nil))
+	})
+
+	t.Run("should return an error when specified local path is not a realm app project", func(t *testing.T) {
```

Comment:
this is _effectively_ the same test as above, but i'm fine with keeping it (to prove out the difference between `LocalPath` not being an app vs `WorkingDirectory` not being an app)